try {
} catch {
  res.status(400).send("Une erreur est survenue");
}
